mission.sqm

```diff
@@ -44562,7 +44562,7 @@ class Mission
 									"STRING"
 								};
 							};
-							value="Gendarmerie Juliett -";
+							value="Gendarmerie Juliet -";
```

Comment:
Huh, it's actually `Juliett` (according to the NATO phonetic alphabet) and we consistently use it wrong throughout F3, aren't we?